src/codegen/expression.rs

```diff
@@ -3200,17 +3243,27 @@ fn code(loc: &Loc, contract_no: usize, ns: &Namespace, opt: &Options) -> Express
     Expression::AllocDynamicBytes(*loc, Type::DynamicBytes, size.into(), Some(code))
 }
 
-fn string_to_expr(string: String) -> Expression {
+fn format_error(string: String) -> Expression {
     Expression::FormatString(
         Loc::Codegen,
-        vec![(
-            FormatArg::StringLiteral,
-            Expression::BytesLiteral(
-                Loc::Codegen,
-                Type::Bytes(string.as_bytes().len() as u8),
-                string.as_bytes().to_vec(),
+        vec![
+            (
+                FormatArg::StringLiteral,
+                Expression::BytesLiteral(
+                    Loc::Codegen,
+                    Type::Bytes(string.as_bytes().len() as u8),
+                    string.as_bytes().to_vec(),
+                ),
+            ),
+            (
+                FormatArg::StringLiteral,
+                Expression::BytesLiteral(
+                    Loc::Codegen,
+                    Type::Bytes(",\n".as_bytes().len() as u8),
+                    ",\n".as_bytes().to_vec(),
+                ),
```

Comment:
Might as well concatenate the last two string literals at compile time rather than runtime.